tools/shared/src/main/scala/org/scalajs/core/tools/sourcemap/SourceMapWriter.scala

```diff
@@ -194,18 +221,26 @@ class SourceMapWriter(
    */
   private def writeBase64VLQ(value0: Int): Unit = {
     // Sign is encoded in the least significant bit
-    var value =
+    val value =
       if (value0 < 0) ((-value0) << 1) + 1
       else value0 << 1
```

Comment:
I wonder whether a branchless equivalent of this would improve things. Assuming `writeBase64VLQ` was still visible in the top of profile data.

*If* the JVM optimizes `Math.abs(int)` as branchless, the following is branchless and does the same thing:
```
val value = (Math.abs(value0) << 1) | (value0 >>> 31)
```
because:
* If `value0 < 0`: `Math.abs(value0) == -value0` and `value0 >>> 31 == 1` so the rhs `== ((-value0) << 1) | 1 == ((-value0) << 1) + 1`
* If `value0 >= 0`: `Math.abs(value0) == value0` and `value0 >>> 31 == 0` so the rhs `== (value0 << 1) | 0 == value0 << 1`

If the JVM does not optimize `abs(int)` as branchless, we have to do it ourselves:
```scala
val signExtended = value0 >> 31
val value = (((value0 ^ signExtended) - signExtended) << 1) | (signExtended & 1)
```
Source for the branchless `abs`: https://hbfs.wordpress.com/2008/08/05/branchless-equivalents-of-simple-functions/